.bazelrc

```diff
@@ -3,6 +3,10 @@ common --enable_bzlmod=true
 common:rbe --java_runtime_version=11
 common:rbe --tool_java_runtime_version=11
 
+common:workspace --enable_bzlmod=false --enable_workspace=true
+#common:workspace --enable_platform_specific_config
```

Comment:
These are commented out, is that intentional?

Reply:
Yeah, tried using a hermetic toolchain. It didn't help.